seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/source/JdbcSourceReader.java

```diff
@@ -60,12 +60,14 @@ public void close() throws IOException {
     public void pollNext(Collector<SeaTunnelRow> output) throws Exception {
         JdbcSourceSplit split = splits.poll();
```

Comment:
move to synchronized code block ->  line#64

Reply:
`splits.poll()` should be after the lock is acquired?

Reply:
done